src/main/java/frc/robot/subsystems/DriveSub.java

```diff
@@ -0,0 +1,102 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismLigament2d;
+import edu.wpi.first.wpilibj.smartdashboard.Mechanism2d;
+
+public class DriveSub extends SubsystemBase {
+
+    private CANSparkMax motor;
+    private SingleJointedArmSim armSim;
+    private Mechanism2d mech2d;
+    private MechanismLigament2d wheelLigament;
+    
+    private boolean isReal = false; // check is real or simulated
```

Comment:
Actually check this with Robot.isReal() instead of passing it in. Passing a value like this around is annoying and hard to maintain.